### DISCUSSION TITLE
Shelly EM ed inversione delle letture

### DISCUSSION THREAD

**Discussion by gianpalm** [indomus/forum#1027] [GENERALE - Discussioni generiche]
Ciao. 
Ho acquistato più di un anno fa uno shelly EM con doppia pinza CT. L'ho connesso alla rete WiFi di casa ma l'ho mantenuto (per scelta) isolato dal loro Cloud e l'ho integrato solo con HA in modo nativo. Tengo entrambi (Shelly e HA) sempre aggiornati per garantire il miglior funzionamento.
Da subito ho notato che le letture dello shelly si invertivano di tanto in tanto, in modo del tutto casuale. Entrambe le letture delle due CT si invertono contemporaneamente.
Ho contatto il supporto (abbastanza efficiente e preciso nella mia esperienza) e mi hanno sostituito per ben due volte lo Shelly EM ma in tutti i casi, appena rimontato, funziona per qualche ora/giorno e poi le letture iniziano ad invertirsi periodicamente.
L'inversione la vedo sia sui grafici di HA sia sulla GUI Web nativa dello Shelly (ed anche ovviamente nei log dello Shelly EM stesso).
In allegato un esempio di un grafico che mostra l'inversione.
![image](https://user-images.githubusercontent.com/68282260/141382806-779b0d1e-d596-4f99-a531-5c5fc3d981a8.png)

Qualcuno ha esperienza di come eliminare questo fastidioso problema (che il supporto dice di non aver mai riscontrato)?
Grazie
Ciao

**Comment by ocimin** [comment]
Hai provato a fare le registrazioni con una sola pinza? Giusto per escludere che le due pinze interferiscano in qualche modo.
